db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+{
+  admin: "Account with administrative rights",
+  moderator: "Moderator role. Allows to update other's posts and comments",
+  editor: "Account is allowed to create own posts, publish them and update them"
```

Comment:
That seems to be good idea. The one thing that concerns me is what if we add more roles through the interface?
But for now it should be ok
@DevMike how do you think?